Makefile

```
@@ -147,6 +147,10 @@ lint-api-fix: $(GOLANGCI_LINT_KAL)
 lint-config: $(GOLANGCI_LINT) ## Verify golangci-lint linter configuration
 	$(GOLANGCI_LINT) config verify
 
+.PHONY: verify-boilerplate
+verify-boilerplate: ## Verify boilerplate text exists in each file
+	 ./hack/verify-all.sh
```

>  ran make verify target, it returns:
> 
> make: *** No rule to make target `verify'. Stop.
> 
> I instead ran the verification script directly:
> ./hack/verify-all.sh
> 
> It produced no output (I think which indicates success).
> Please let me know if anything else is needed.

That's because this Makefile target needs updating to 


```suggestion
.PHONY: verify
verify:
	 ./hack/verify-all.sh
```